documentation/examples/array_comprehensions.coffee

```diff
@@ -1,3 +1,7 @@
+# Methods
+eat = (food) -> "#{food} eaten."
```

Comment:
I feel like this line should be just above the `eat food` line, to keep it together.